src/components/custom/_signup-public.js

```diff
@@ -290,7 +291,7 @@ const SignupPublic = ({
             </SignupFormWrapper>
             <BackgroundWrapper direction="column" ai="center">
                 <LinkFlex ai="center" onClick={redirectToDerivApp}>
-                    <StyledHeader font_size="2.8rem" width="28.2rem" align="center" color="grey-8">
+                    <StyledHeader size="2.8rem" max_width="28.2rem" align="center" color="grey-8">
```

Comment:
Should be `mw` here?
```suggestion
                    <StyledHeader size="2.8rem" mw="28.2rem" align="center" color="grey-8">
```

Reply:
but in other places its `max_width`, should we stick with `mw`?

Reply:
it should be mw, let me check

Reply:
wait, if max_width is mw, then what is min_width?

Reply:
🔢 true, better stick with max_width and min_width